src/app/contact/contact.component.ts

```diff
@@ -1,10 +1,29 @@
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
+import { homePageService } from '../services/homepage.service';
 @Component({
-    selector:'app-contact',
-    templateUrl:'./contact.component.html',
-    styleUrls:['./contact.component.scss']
+    selector: 'tf-contact',
+    templateUrl: './contact.component.html',
+    styleUrls: ['./contact.component.scss']
 })
-export class ContactComponent{
-    constructor(){}
-    ngOnInit(){}
-}
\ No newline at end of file
+export class ContactComponent implements OnInit {
+    demo: String;
+    demoDescription: String;
+    demoEmail: String;
+    support: String;
+    supportDescription: String;
+    supportEmail: String;
+    constructor(public contactresponse: homePageService) {}
+    ngOnInit () {
+        this.contactresponse.getcontactresponse().subscribe(
+            (response) => {
+                this.demo = response.json().items[0].fields.contactType;
```

Comment:
do not use hard coded value in array. apply the for each condition for items array.